From 399d38782d39334b657893c4e157d1bbc305e084 Mon Sep 17 00:00:00 2001
From: genius user <genius@ReliableEmbeddedSystems.com>
Date: Tue, 24 Mar 2015 22:17:09 +0000
Subject: [PATCH] allow only 1 build at a time

Signed-off-by: genius user <genius@ReliableEmbeddedSystems.com>
---
 yocto-controller/controller.cfg.example | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/yocto-controller/controller.cfg.example b/yocto-controller/controller.cfg.example
index e5d452f..075bac6 100644
--- a/yocto-controller/controller.cfg.example
+++ b/yocto-controller/controller.cfg.example
@@ -47,7 +47,7 @@ c['buildbotURL'] = "http://localhost:8010"
 #                     BuildWorker("builder2", "<PASS>", max_builds=3),]
 
 from buildbot.buildslave import BuildSlave as BuildWorker
-c['workers'] = [BuildWorker("example-worker", "<PASS>", max_builds=3),]
+c['workers'] = [BuildWorker("example-worker", "<PASS>", max_builds=1),]
 
 # 'workerPortnum' defines the TCP port to listen on for connections from 
 # workers. This must match the value configured into the buildworkers 
-- 
1.9.1

